remediation_worker/jobs/rds_remove_public_endpoint/README.md

```diff
@@ -0,0 +1,74 @@
+# Private rds snapshot
```

Comment:
Disable public access to RDS instances

Reply:
the description is below
This job disables public access to RDS instances to to ensure it is not connected by any internet hosts.

Reply:
I'm asking you to change the job title from Private rds snapshot to Disable public access to RDS DBInstances.